packages/nc-gui/lib/form.ts

```
@@ -20,28 +23,440 @@
+                      break
+                  }
+                }
+              }
+            }
+          } else {
+            switch (typeof filter.value) {
+              case 'boolean':
+                val = !!this.formState[field]
+                break
+              case 'number':
+                val = isNaN(parseFloat(this.formState[field])) ? this.formState[field] : +this.formState[field]
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)